redshift.tf

```diff
@@ -6,5 +6,6 @@ resource "aws_redshift_cluster" "redshiftEncryptedFalse" {
   node_type          = "dc1.large"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure Redshift Cluster is not Publicly Accessible**
 
**Vulnerability**
Ensure Redshift clusters are not publicly accessible to minimise security risks.
 
**Remediation**
When your Amazon Redshift clusters are publicly accessible and have a public IP address, every machine on the Internet can establish a connection to your clusters and this can increase the opportunity for malicious activity such as SQL injections or Distributed Denial of Service (DDoS) attacks.